packages/blocks/src/embed-linked-doc-block/commands/index.ts

```diff
@@ -0,0 +1,7 @@
+import type { Command } from '@blocksuite/block-std';
+
+import { insertLinkByQuickSearchCommand } from './insert-link-by-quick-search.js';
+
+export const commands: Partial<Record<keyof BlockSuite.Commands, Command>> = {
```

Comment:
Maybe add a helper type for this?

Reply:
Thanks for your comment! A helper type have been added.